components/docs/pluginsList.tsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from "react";
+import { Table } from "react-bootstrap";
+const url = "https://registry.npmjs.com/-/v1/search";
+
+interface NPMPackage {
+  name: string;
+  scope: string;
+  version: string;
+  description: string;
+  date: string;
+  links: {
+    npm: string;
+    homepage: string;
+    repository: string;
+    bugs: string;
+  };
+}
+
+const ignoredPlugins = ["@grouparoo/ui-components", "@grouparoo/client-web"];
+
+export default function PluginsList() {
```

Comment:
Clever approach!

Reply:
Heh, I really wanted something fancier than a table, but I couldn't come up with a good idea without having access to any assets (icons) as well